roles/uperf/templates/server_vm.yml.j2

```diff
@@ -55,12 +55,10 @@ spec:
         chpasswd: { expire: False }
         runcmd:
 {% if workload_args.server_vm.network.multiqueue.enabled %}
-          - yum install -y ethtool
+          - dnf install -y ethtool
           - ethtool -L eth0 combined {{ workload_args.server_vm.network.multiqueue.queues }}
 {% endif %}
-          - yum install -y yum-plugin-copr
-          - yum copr enable ndokos/pbench -y
-          - yum install -y pbench-uperf redis git
+          - dnf install -y uperf redis
```

Comment:
Do we need to enable the appstream?

Reply:
No, it's in the default repositories. (tested it locally ;) )

Reply:
nice!